ThingIFSDK/ThingIFSDK/TargetThing.swift

```diff
@@ -28,4 +28,23 @@ public extension TargetThing {
             return self.typedID.id
         }
     }
+
+}
+
+internal func makeTargetThing(
+  _ json: [String : Any],
+  layoutPosition: LayoutPosition,
+  vendorThingID: String? = nil) throws -> TargetThing
+{
+    var jsonWithVendorThingID = json
```

Comment:
@satoshikumano Sorry, I misunderstood. arguments of function is constant so we can not directory modify it. Though, We can use same name variable for argument. I renamed `jsonWithVendorThingID` to `json`.